CHANGELOG/README.md

```diff
@@ -1,6 +1,6 @@
 # Production recommendation
 
-The minimum recommended etcd versions to run in **production** are 3.3.18+, 3.4.2+, v3.5.3+. Refer to the [versioning policy](https://etcd.io/docs/v3.5/op-guide/versioning/) for more details.
+The minimum recommended etcd versions to run in **production** are 3.4.2+ and v3.5.3+. Refer to the [versioning policy](https://etcd.io/docs/v3.5/op-guide/versioning/) for more details.
```

Comment:
There is a known data inconsistent issue (https://github.com/etcd-io/etcd/issues/11651) in both 3.4, and fxied in 3.4.8.  Although the data inconsistent issue (https://github.com/etcd-io/etcd/issues/13766) in 3.5 was fixed in 3.5.3, but there is a regression which is fixed in 3.5.4. 

So I would suggest to change the minimum recommended etcd version to 3.4.8+ and 3.5.4+.

Reply:
@ahrtr ok, that sounds good. Thanks!